build/ci/community/clean_demo_pr.sh

```diff
@@ -1,15 +1,13 @@
 #!/bin/bash -xe
 set +o pipefail
 
-sleep 20
-
 print_checks_and_delete_pr(){
   community_operators_branch=$1
   cluster_kind=$2
   export repo_pr=`gh pr list --repo $github_csiblock_community_operators_repository | grep $community_operators_branch`
   if [[ "$repo_pr" == *"$community_operators_branch"* ]]; then
     echo "The $cluster_kind checks:"
-    gh pr checks $community_operators_branch --repo $github_csiblock_community_operators_repository
+    gh pr checks $community_operators_branch --repo $github_csiblock_community_operators_repository || true
```

Comment:
AFAIK the `set` command is here for that. Let's try to not abuse the `||` operator

Reply:
I want that only on this line the script won't exist because of a fail, because it just showing information, so I don't want to change the `set` that will change the behavior of the rest of the script

Reply:
You can change it back

Reply:
what do you mean?

Reply:
I suggest reading about the `-`/`+` flag prefixes

Reply:
is it really better to set and unset a global state just for one command?

Reply:
@zingero ?

Reply:
@zingero gentle reminder

Reply:
Oh, sorry. I thought we're waiting for @matancarmeli7.
We can wrap only this command with the `set` command.
Which do you folks prefer?

Reply:
` || true` is shorter than two additional lines, but indeed hacky.
I think the current project convention is actually the shorter [`|| :`](https://github.com/IBM/ibm-block-csi-driver/blob/0f5ff540d6d8e69018201db2d00b34128fd3bae3/scripts/ci/build_push_images.sh#L27).
I prefer to stick to the convention, and if anyone wants, they can open a ticket to change the convention throughout the project